apps/qubit/config/filters.yml

```diff
@@ -25,5 +22,8 @@ QubitMeta:
 QubitLimitResults:
   class: QubitLimitResults
 
+QubitTransaction:
+  class: QubitTransactionFilter
+
```

Comment:
This filters can be extended/overwritten in the plugins, we should make sure those reference `QubitTransaction` now. I'm not sure if the reordering affects those extensions too but we should check that out too if the order is important.

Reply:
Huh, okay I'll have a look at the plugins for filter config.